frontend/src/views/Lobby.tsx

```diff
@@ -59,10 +61,15 @@ function LobbyPage() {
     setRoomId(room.roomId);
     setActive(room.active);
     setDifficulty(room.difficulty);
+    setDuration(room.duration / 60);
   };
 
+  // Function to determine if the given user is the host or not
+  const isHost = useCallback((user: User | null) => user?.nickname === host?.nickname, [host]);
```

Comment:
On PR #88, I [replaced the nickname comparison with `userId` comparison](https://github.com/rocketden/main/pull/88/files#diff-0e48089598b398a1fce461bcccf5ff61693ec954880e97a02b2ae39c7cb8a31cR334) -- just heads up, since there will be merge conflict issues here, and I might suggest comparing `userId`s instead because that seems like a more certain system long-term.

This does obviously come with the caveat that we have to determine how to hide `userId`s on the frontend, so there's a good argument for comparing by nickname, but either way we should keep it consistent by modifying #88 (the other PR) or #92 (this PR). (This same idea applies elsewhere for "nickname vs. `userId`" in the PR.)

Reply:
Good catch, I updated all instances to use userId now